csrc/ir/utils.h

```diff
@@ -66,13 +66,17 @@ struct MatmulInputs {
   Val* bias = nullptr;
   Val* alpha = nullptr;
   Val* beta = nullptr;
+  MmaLayout mma_layout;
 };
 
 //! Matches the following matmul patterns.
 //! Matmul: A x B, alpha * A x B
 //! Matmul + Bias: A x B + C,  alpha * A x B + C, A x B + beta * C,
 //!   alpha * A x B  + beta * C
-//! Note: For simplicity, we assume the MmaOp to be in the first operand.
+//! Assumptions:
+//! 1. For simplicity, we assume the MmaOp to be in the first operand.
+//! 2. If mma layout is TN ([M, K], [N, K]), alpha, beta parameters are ignored
```

Comment:
Can you expand in a comment on the reasoning here? I thought we were going to match for a transpose on the B operand instead of looking at layout.

Reply:
I think based on our last meeting on Thursday, we had decided that we don't need a transpose.  We just need to setup the fusedMultiplySum with tensors of appropriate shapes and plumb it down to at::linear. Please correct me if I am wrong. cc @kevinstephano.